### PR TITLE
Feature/picklist

### PR DESCRIPTION
**Pull request by paulfalgout**
- [x] Add the Clubhouse Story ID: [ch20496]

### CODE REVIEW

File: src/js/behaviors/picklist-transport.js [hunk 1]
```diff
@@ -113,7 +106,7 @@ export default Behavior.extend({
     const nextIndex = $items.index($highlighted) + 1;
 
     if (nextIndex === $items.length) {
-      if (!this.shouldLoopList()) return $highlighted;
+      if (!this.getOptions('shouldLoopList')) return $highlighted;
```
Comment (luketlancaster) [inline src/js/behaviors/picklist-transport.js]: getting a `this.getOptions is not a function` error when trying to loop the list manually via arrow keys here

Comment (paulfalgout) [reply src/js/behaviors/picklist-transport.js]: Should be getOption

Comment (paulfalgout) [reply src/js/behaviors/picklist-transport.js]: Now that I'm looking at this again.. this is an older option copied over.  I have no doubt that it works, but currently there's no way to use it, and we never had and may never have a use case.  In this case I'd be fine with removing the `shouldLoopList` option altogether and simplify the whole bit.

Comment (luketlancaster) [reply src/js/behaviors/picklist-transport.js]: So behavior would be to not loop the list at all. If you hit the bottom and hit the down arrow, you don't go anywhere. Same with the top and the up arrow?

Comment (paulfalgout) [reply src/js/behaviors/picklist-transport.js]: correct

**Comment by luketlancaster** [comment]
man...i really should have done a merge instead of a rebase there. old habits die hard though

**Pull request merged**